html-forms/html-forms/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form>
+      <p>Email</p>
+      <label for="email"></label>
+      <input type="email" name="email" id="email" />
```

Comment:
There should be text inside your `label` on line 14. I would just remove your `p` element and throw that text inside the `label`